<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Frame</class>
 <widget class="QFrame" name="Frame">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>424</width>
    <height>341</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Charts</string>
  </property>
  <property name="windowIcon">
   <iconset resource="project51.qrc">
    <normaloff>:/images/circle.png</normaloff>:/images/circle.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">#rbtnf{
background:rgb(255, 172, 76);
}
#rbtnm{
background:rgb(96, 205, 255);
}
#rbtnb{
background:rgb(121, 255, 175);
}
#rbtnf:hover{font-weight:bold}
#rbtnm:hover{font-weight:bold}
#rbtnb:hover{font-weight:bold}
QComboBox{
background-color:white;
border: none;
border-radius: 4px;
}
QRadioButton
{color:white;
}
QLabel
{color:white;
}
#Frame
{background:#333;
}
QPushButton
{background:white;
color:black;
border-radius:15px;
}
QPushButton:hover
{color:black;
border-radius:15px;
background:rgb(159, 255, 157);
}
QTextEdit
{background:rgb(255, 255, 255);
border-radius:15px;
}
QLineEdit
{background:transparent;
border:none;
color: white;
border-bottom:1px solid #e2e2e2;
}</string>
  </property>
  <widget class="QPushButton" name="btnshare">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>140</y>
     <width>141</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Show MarketShare</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnbest">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>140</y>
     <width>141</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Best Selling Brand</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnshow">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>220</y>
     <width>141</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>More Charts</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnback">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>220</y>
     <width>41</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="project51.qrc">
     <normaloff>:/images/back-arrow-1767531-1502435.png</normaloff>:/images/back-arrow-1767531-1502435.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>28</width>
     <height>28</height>
    </size>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>40</y>
     <width>291</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="text">
    <string>ðŸ“ˆ Growth Charts ðŸ“Š</string>
   </property>
  </widget>
  <widget class="QPushButton" name="rbtnb">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>220</y>
     <width>61</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Calibri Light</family>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="text">
    <string>$</string>
   </property>
  </widget>
  <widget class="QPushButton" name="rbtnm">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>220</y>
     <width>61</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Calibri Light</family>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="text">
    <string>$$</string>
   </property>
  </widget>
  <widget class="QPushButton" name="rbtnf">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>220</y>
     <width>61</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Calibri Light</family>
     <pointsize>9</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>$$$</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>270</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="text">
    <string> Flagship</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>270</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Mid-range</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>270</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="text">
    <string> Budget</string>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="project51.qrc"/>
 </resources>
 <connections/>
</ui>
